src/screens/map-screen/map-screen.component.tsx

```diff
@@ -246,19 +275,39 @@ const MapScreen = () => {
    */
   const animateToCurrentLocation = () => {
     if (currentLocation)
-      mapRef.current.animateToRegion({
-        latitude: currentLocation.latitude,
-        longitude: currentLocation.longitude,
+      animateToRegion({
+        ...currentLocation,
         latitudeDelta: currentRegion.latitudeDelta,
         longitudeDelta: currentRegion.longitudeDelta,
       });
   };
 
+  /**
+   * Sets the indoor route so steps can be displayed
+   */
+  const onStartTravelPlan = () => {
+    setFloorPaths(
+      IndoorPathPlanningService.getInstance().updateFloorPaths(
+        startLocation,
+        endLocation,
+        travelMode === TravelMode.ACCESSIBLE
+      )
+    );
+  };
+
+  const isTravelling = (): boolean => {
+    return travelState === TravelState.TRAVELLING;
+  };
+  const isPlanning = (): boolean => {
+    return travelState === TravelState.PLANNING;
+  };
+
+  const isTravelStateNone = (): boolean => {
+    return travelState === TravelState.NONE;
+  };
```

Comment:
Thanks for doing this but now the burden is on you to ensure it's used everywhere :(. Please find and replace everywhere that stil uses the old method to check the travelstate.

Reply:
Because we have those comparisons all over the map-screen, I wanted to centralize them and have one method so it can be clear to others. It's not meant to be exported or passed to other component since it will break test.

Reply:
Disagree, we shouldn't be checking it one way i one component and another one in another. It's also only used in one other component. I don't think this is a big enough reason to block though